haystack/document_store/elasticsearch.py

```diff
@@ -3,7 +3,7 @@
 import time
 from copy import deepcopy
 from string import Template
-from typing import List, Optional, Union, Dict, Any
+from typing import List, Optional, Union, Dict, Any, Generator
```

Comment:
I think `Generator` is not being used in this file.

Reply:
We were referring to a native python generator here (not RAG etc) :)

Reply:
Thanks for spotting that. This is a leftover from trying to pass a generator to `write_documents` and `write_labels`. I will remove this.